docs/source/user-manual/links-to-item.md

```diff
@@ -19,6 +19,9 @@ For this situation, you can visit the {guilabel}`Links and references` page, whi
 An item references another item by creating a hyperlink to this item, using the item in a block, referencing it in the {guilabel}`Related Items` field, or referencing it in a relation field.
 ```
 
+```{versionadded} Volto 17.0.0-alpha.19
```

Comment:
The version* admonitions should be below the headings to which they refer, because when people click anchors, the version* admonitions will scroll out of view.

````suggestion
```{versionchanged} Volto 17.0.0-alpha.19
This feature is now enabled by default.
```

```{versionadded} Volto 16.?.?
This feature was added in Volto 16.?.?, but it was disabled by default.
````

Reply:
Oopsie! 16 admonition should come before 17 admonition!

Reply:
Edited suggestion.

Reply:
@stevepiercy This info is relevant to the feature covered by this entire page. Should it go just under the h1 heading?

Reply:
@stevepiercy I applied your suggestion manually so I could add the right version for 16. Thanks.